src/LinuxTracing/UprobesUnwindingVisitor.cpp

```diff
@@ -226,6 +226,14 @@ void UprobesUnwindingVisitor::Visit(uint64_t event_timestamp,
                                    event_data.data.get()};
   std::vector<StackSliceView> stack_slices{event_stack_slice};
 
+  const auto& stream_id_to_user_stack = thread_id_stream_id_to_stack_slices_.find(event_data.tid);
+  if (stream_id_to_user_stack != thread_id_stream_id_to_stack_slices_.end()) {
+    for (const auto& [unused_stream_id, user_stack_slice] : stream_id_to_user_stack->second) {
+      stack_slices.emplace_back(user_stack_slice.start_address_, user_stack_slice.size_,
+                                user_stack_slice.data_.get());
```

Comment:
The final underscore of the fields of `StackSlice` should be remove as they are public fields of a `struct`.